cmd/cluster/core/dump.go

```diff
@@ -765,3 +788,19 @@ func shouldDumpKubevirt(nodePools []*hyperv1.NodePool) ([]kubevirtExtCluster, bo
 
 	return kubevirtInExternalInfras, localKubevirtInUse
 }
+
+func isResourceRegistered(discoveryClient discovery.DiscoveryInterface, gvk schema.GroupVersionKind) (bool, error) {
```

Comment:
can we unit test this? see e.g. https://github.com/kubernetes/autoscaler/blob/4d0511363eeeff657119797dd8d26e851dcc3459/cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller_test.go#L123C2-L123C17

Reply:
@enxebre Yup. Thanks for the example. Updated.